src/scripts/components/Component.ts

```diff
@@ -0,0 +1,62 @@
+
+
```

Comment:
Prázdné řádky?! (Prettier by tě mohl osvobodit.)

Reply:
To je v pohodě počet prázdných řádků.